### PR TITLE
(Temporarily) Remove multiple reactions from #index

### LINKED ISSUE #19134: Multiple Reactions: Update index page to display [current] heart icon + reaction count on Posts
This is a change needed in order to launch, but this separate followup issue (to be addressed post-launch) is focused on fixing the actual issue.

### Problem Description

Currently on index pages, there is an inconsistency on how reactions are displayed on the list of Posts. This was first noticed [here](https://github.com/forem/forem/issues/19064#issuecomment-1421240035) and [here](https://github.com/forem/forem/issues/19064#issuecomment-1424891619). 

Image

### Desired adjustment needed for launch
This is not the fix, but the temporary display tweak. Showing the old reaction display (heart icon version) instead of two versions will give us more flexibility to address the root cause separately without blocking the launch.

Please adjust the index page posts such that it follows this look and feel:

Image

### PR DESCRIPTION
**Pull request by jaw6**
## What type of PR is this? (check all applicable)

- [ ] Refactor
- [ ] Feature
- [ ] Bug Fix
- [ ] Optimization
- [ ] Documentation Update
- [x] Other

## Description

We're still seeing inconsistent behavior from the `multiple_reactions` feature-flag on `#index` views. For initial launch, we're going to remove `multiple_reactions` from `#index` views, restoring the original/current "simple reactions" view. We have a separate issue (#19135) to revert this change, when we're ready to turn `multiple_reactions` on across all `#index` views.

## Related Tickets & Documents

- Closes #19134
- Related Issue #19135

## Added/updated tests?

- [ ] Yes
- [x] No, and this is why: this isn't tested
- [ ] I need help with writing tests

## [Forem core team only] How will this change be communicated?

_Will this PR introduce a change that impacts Forem members or creators, the
development process, or any of our internal teams? If so, please note how you
will share this change with the people who need to know about it._

- [ ] I've updated the [Developer Docs](https://developers.forem.com) or
      [Storybook](https://storybook.forem.com/) (for Crayons components)
- [ ] This PR changes the Forem platform and our documentation needs to be
      updated. I have filled out the
      [Changes Requested](https://github.com/forem/admin-docs/issues/new?assignees=&labels=&template=changes_requested.md)
      issue template so Community Success can help update the Admin Docs
      appropriately.
- [ ] I've updated the README or added inline documentation
- [ ] I've added an entry to
      [`CHANGELOG.md`](https://github.com/forem/forem/tree/main/CHANGELOG.md)
- [ ] I will share this change in a [Changelog](https://forem.dev/t/changelog)
      or in a [forem.dev](http://forem.dev) post
- [x] I will share this change internally with the appropriate teams
- [ ] I'm not sure how best to communicate this change and need help
- [ ] This change does not need to be communicated, and this is why not: _please
      replace this line with details on why this change doesn't need to be
      shared_

## [optional] What gif best describes this PR or how it makes you feel?

alt_text

### CODE REVIEW

**Review by rt4914 — APPROVED** [review]
LGTM, Thanks.

**Review by benhalpern — APPROVED** [review]

**Comment by jaw6** [comment]
Putting this on hold until after #19157 merges

**Comment by mirie** [comment]
> Putting this on hold until after #19157 merges

Spoke with @benhalpern and @pkfrank today during our sync. #19157 is not necessary for launch -- can be a post-launch change. This PR however _**is necessary**_ for launch, so moving forward with this one.

Target is not to launch today but to be ready for tomorrow. As this has all the approvals and there's no need to wait on merging this one, will go ahead and move out of draft PR and merge this in.

**Pull request merged**
